drivers/pwm/pwm-sophgo-sg2042.c

```diff
@@ -93,6 +90,69 @@ static int pwm_sg2042_apply(struct pwm_chip *chip, struct pwm_device *pwm,
 	return 0;
 }
 
+static void pwm_sg2044_enable(struct sg2042_pwm_ddata *ddata, struct pwm_device *pwm, bool enabled)
+{
+	u32 pwm_value;
+
+	pwm_value = readl(ddata->base + REG_PWMSTART);
+
+	if (enabled)
+		writel(pwm_value | BIT(pwm->hwpwm), ddata->base + REG_PWMSTART);
+	else
+		writel(pwm_value & ~BIT(pwm->hwpwm), ddata->base + REG_PWMSTART);
+}
+
+static void pwm_sg2044_outputenable(struct sg2042_pwm_ddata *ddata, struct pwm_device *pwm,
+				    bool enabled)
```

Comment:
`enable` should be `output`. As it is a direction hint.  the function name should be also changed.